examples/plot_glm_retinal_ganglia.py

```diff
@@ -52,23 +52,33 @@
 from scipy.linalg import hankel
 
 from pyglmnet import GLM
+from pyglmnet.datasets import fetch_rgc_data
 
 import matplotlib.pyplot as plt
 from tempfile import TemporaryDirectory
 
+
+def read_json(dpath, file_name):
+    """Function to read JSON file from a given ``dpath``"""
+    with open(op.join(dpath, file_name), 'r') as f:
+        dataset = json.loads(f.read())
+    return dataset
+
+
 ########################################################
 #
 # Fetch the dataset. The JSON file contains the
 # following keys:  ``stim`` (binary stochastic stimulation),
 # ``stim_times`` (time of the stimulation), and
 # ``spike_times`` (recorded time of the spikes)
 
-from pyglmnet.datasets import fetch_rgc_data
-
-with TemporaryDirectory(prefix="tmp_glm-tools") as temp_dir:
-    dpath = fetch_rgc_data(dpath=temp_dir, accept_rgcs_license=True)
-    with open(op.join(dpath, 'data_RGCs.json'), 'r') as f:
-        rgcs_dataset = json.loads(f.read())
+# use data if locally downloaded, else ask for license agreement
+if op.isdir("glm-data") and op.exists("glm-data/data_RGCs.json"):
+    rgcs_dataset = read_json("glm-data", 'data_RGCs.json')
+else:
+    with TemporaryDirectory(prefix="tmp_glm-tools") as temp_dir:
+        dpath = fetch_rgc_data(dpath=temp_dir, accept_rgcs_license=False)
+        rgcs_dataset = read_json(dpath, 'data_RGCs.json')
```

Comment:
this is too complex for example. You should do some magic in the dataset fetcher.

See how MNE does it, search for "get_config" function. Or scikit-learn does it more simply by using environment variables: https://github.com/scikit-learn/scikit-learn/blob/72b3041ed57e42817e4c5c9853b3a2597cab3654/sklearn/datasets/_base.py#L30